lib/pharos/host/ubuntu/scripts/reset.sh

```diff
@@ -6,12 +6,14 @@ systemctl stop kubelet
 systemctl disable kubelet
 
 if systemctl is-active --quiet docker ; then
+    # shellcheck disable=SC2046
     docker stop $(docker ps -q)
     systemctl stop docker
     systemctl disable docker
 fi
 
 if systemctl is-active --quiet crio ; then
+    # shellcheck disable=SC2046
```

Comment:
the prefered way would be to turn something like `crictl stopp $(crictl pods -q)` into something that first reads the subcommand result into an array and then uses that array as an argument, but i suppose we can assume here that the `-q` outputs are glob-safe and do not include spaces.